test/Directory.Build.props

```diff
@@ -2,7 +2,6 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <DefaultTestTargetFramework>netcoreapp2.1</DefaultTestTargetFramework>
```

Comment:
What if I want to just run one test project with dotnet test? I think DefaultTestTargetFramework should be defaulted to netcoreapp2.1 whenever it is not set.

```xml
<DefaultTestTargetFramework Condition="'$(DefaultTestTargetFramework)' == ''">netcoreapp2.1</DefaultTestTargetFramework>
<DefaultTestTargetFramework Condition="'$(UseIntrinsics)' == 'true' AND '$(DefaultTestTargetFramework)' == ''">netcoreapp3.0</DefaultTestTargetFramework>
```

Something like that maybe?